src/main/java/gov/usds/case_issues/controllers/errors/ApiControllerAdvice.java

```diff
@@ -17,6 +17,13 @@
 
 	private static final Logger LOG = LoggerFactory.getLogger(ApiControllerAdvice.class);
 
+	@ExceptionHandler
+	@ResponseStatus(HttpStatus.BAD_REQUEST)
+	public SpringRestError handleIllegalArgumentException(IllegalArgumentException e, HttpServletRequest req) {
+		LOG.debug("Illegal argument of: {}", e);
```

Comment:
Oh, this is an interesting slf4j question—does it actually log with the format you're looking for, or does it log the literal string `{}` and then print the exception stack trace?